deploy/charts/wildside/templates/deployment.yaml

```
@@ -0,0 +1,115 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "wildside.fullname" . }}
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)